README.markdown

```diff
@@ -6,6 +6,42 @@ standards such as [JSON Schema](http://json-schema.org), [JSON
 Pointer](https://www.rfc-editor.org/rfc/rfc6901),
 [JSONL](https://jsonlines.org), and more.
 
+Example
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()  {
+    // Creating JSON using parse
+    sourcemeta::jsontoolkit::JSON document =
```

Comment:
On the README we likely have more horizontal space, so you can format this a bit nicer:

```c++
sourcemeta::jsontoolkit::JSON document = sourcemeta::jsontoolkit::parse(R"JSON({
  "name": "John Doe",
  "age": 20,
  "address": "zxy"
})JSON");
```

Reply:
@tony-go It'd have been so cool to have  https://github.com/sourcemeta/jsontoolkit/issues/506 here 😅

Reply:
Yeah, I do bad dreams because of this hahaha :)